orttraining/orttraining/test/gradient/gradient_op_test_utils.cc

```diff
@@ -170,6 +215,7 @@ void GradientOpTester::Run(
           }
           if (!valid)
             break;
+          } else {
```

Comment:
Something seems to have gone wrong with the formatting here (and with the added "`}`" on L233). Is this leftover debugging code that could be reverted?

Reply:
Not sure what is going on with the empty else statement I will remove it.